includes/admin.form.inc

```diff
@@ -40,6 +40,17 @@ function islandora_manuscript_admin_settings_form(array $form, array &$form_stat
           array('pdf', 'image', 'ocr'))),
       ),
     ),
+    // Metadata display.
+    'metadata_display_fieldset' => array(
+      '#type' => 'fieldset',
+      '#title' => t('Metadata display'),
+      'islandora_collection_metadata_display' => array(
```

Comment:
Given how [`system_settings_form`](https://api.drupal.org/api/drupal/modules%21system%21system.module/function/system_settings_form/7.x), more importantly the [submit handler](https://api.drupal.org/api/drupal/modules%21system%21system.module/function/system_settings_form_submit/7.x) works these entries are what the variable names become. That is, this entry is the already existing `islandora_collection_metadata_display` this should be something unique to this module like `islandora_manuscript_metadata_display`.